andela/krypton-ah-backend
# #162414162 Reset Password via email

#### What does this PR do?
Sends Reset Link to user email

#### Description of Task to be completed?
- Send reset link to user email
- Confirm the reset link gotten from user email

#### How should this be manually tested?
- Register on sendgrid and get API key form them
- Add API key to environment variable using variable name SENDGRID_API_KEY
- Clone this branch 
- Run npm install
- Send post request in the format
	{
	  "email":"jeosmith@myserver.com",
	}
- If successful you should see a reply like the below
{
    "success": true,
    "message": "Reset password mail sent successfully"
}
- If not successful you should see a reply like the below
{
    "success": false,
    "message": "Reset password mail not sent"
}

- If successful go to your email (if not in inbox check spam)
- Click the reset link seen
- You will get a response telling you if the link is:
	- Valid
	- Invalid
	- Expired

#### What are the relevant pivotal tracker stories?
PT Link: https://www.pivotaltracker.com/story/show/162414162

#### Screenshots
<img width="1082" alt="screen shot 2019-01-18 at 10 39 10 am" src="https://user-images.githubusercontent.com/32283153/51378411-80be7100-1b0d-11e9-8a48-b811102a3afb.png">
